puppet/lib/puppet/util/puppetdb/config.rb

```diff
@@ -14,6 +14,8 @@ def self.load(config_file = nil)
         :soft_write_failure        => false,
```

Comment:
Ah, it's actually subtly different. I'll leave the code in and just change the config code.